wp-includes/components/class-wp-service-worker-configuration-component.php

```diff
@@ -72,19 +72,24 @@ public function get_script() {
 		);
 		$script .= sprintf( "workbox.setConfig( %s );\n", wp_service_worker_json_encode( $options ) );
 
+		// Vary the prefix by the root directory of the site to ensure multisite subdirectory installs don't pollute each other's caches.
+		$prefix = sprintf(
+			'wp-%s',
+			wp_parse_url( WP_Service_Workers::SCOPE_FRONT === $current_scope ? home_url( '/' ) : site_url( '/' ), PHP_URL_PATH )
+		);
+
 		$cache_name_details = array(
-			// Vary the prefix by the root directory of the site to ensure multisite subdirectory installs don't pollute each other's caches.
-			'prefix'   => sprintf(
-				'wp-%s',
-				wp_parse_url( WP_Service_Workers::SCOPE_FRONT === $current_scope ? home_url( '/' ) : site_url( '/' ), PHP_URL_PATH )
-			),
+			'prefix'   => $prefix,
 			// Also precache name by scope (front vs admin) so that different assets can be precached in each respective application.
 			'precache' => sprintf( 'precache-%s', WP_Service_Workers::SCOPE_FRONT === $current_scope ? 'front' : 'admin' ),
 			'suffix'   => 'v1',
 		);
 
 		$script .= sprintf( "workbox.core.setCacheNameDetails( %s );\n", wp_service_worker_json_encode( $cache_name_details ) );
 
+		// Export the prefix as a constant since it is not accessible via workbox.core.cacheNames, unfortunately (yet).
+		$script .= sprintf( 'const WP_SERVICE_WORKER_CACHE_PREFIX = %s;', wp_service_worker_json_encode( $prefix ) );
```

Comment:
Just double-checking, can the cache name prefix contain special characters such as slashes? Typically the subdirectory in a multisite is just a single folder, but that can be different. _If_ there is a restriction in that regard, we should sanitize the prefix to replace such characters (for example with dashes or underscores).

Reply:
Slashes haven't caused a problem yet. The runtime and precache caches have worked fine with the `/` in the name.